QuickOSM/test/test_connexion_nominatim.py

```diff
@@ -0,0 +1,97 @@
+"""Tests for Nominatim API requests."""
+
+from qgis.testing import unittest
+
+from QuickOSM.core.api.nominatim import Nominatim
+from QuickOSM.core.exceptions import NominatimAreaException, NominatimException
+from QuickOSM.definitions.nominatim import NOMINATIM_SERVERS
+
+__copyright__ = 'Copyright 2019, 3Liz'
+__license__ = 'GPL version 3'
+__email__ = 'info@3liz.org'
+
+
+class MyTestCase(unittest.TestCase):
+    def test_request(self):
+        """Test request.
+
+        This is test is using internet.
+        """
+        nominatim = Nominatim(NOMINATIM_SERVERS[0])
+
+        try:
+            nominatim.query('place')
+        except NominatimAreaException:
+            self.assertFalse(True)
+        except NominatimException:
+            self.assertFalse(True)
+        else:
+            self.assertTrue(True)
```

Comment:
I know you made a copy/paste, but it wasn't good.

It is possible to check for execptions in tests.

Use https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises
Or https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex